engine-standalone-tracing/src/sputnik.rs

```diff
@@ -0,0 +1,79 @@
+use std::cell::RefCell;
+use std::ptr::NonNull;
+use std::rc::Rc;
+
+// TODO: implement evm* tracing traits for `crate::trace::Logs`, and tie this `traced_call` function to the FFI.
```

Comment:
Note TODO

Reply:
Trait implementations are done in #384 

Hooking in with the main standalone binary will be done in a future PR (pending the decision on how the relayer will interact with standalone)